docker/launch_spark_history_server_locally.sh

```diff
@@ -1,20 +1,92 @@
-#!/bin/sh
+#!/bin/sh 
+set -e
 # ENTER AWS CREDENTIALS TO RUN THE DOCKER IMAGE LOCALLY
 
-DOCKER_IMAGE="varabonthu/spark-web-ui:1.0.6"
-LOG_DIR="s3a://<bucket_name>/<spark-event-logs-prefix>/"
-ENDPOINT="s3.<region>.amazonaws.com"
+CLASS="org.apache.spark.deploy.history.HistoryServer"
+AWS_REGION="${4:-us-east-1}"
+ENDPOINT="s3.$AWS_REGION.amazonaws.com"
+CONTAINER_NAME="${5:-spark-history-server}"
+do_start(){
+	S3_BUCKET=$1
+	S3_BUCKET_PREFIX=$2
+	LOG_DIR="s3a://$S3_BUCKET/$S3_BUCKET_PREFIX"
+	DOCKER_IMAGE="$USER/spark-web-ui:latest"
 
-AWS_ACCESS_KEY_ID=""
-AWS_SECRET_ACCESS_KEY=""
-AWS_SESSION_TOKEN=""
```

Comment:
We need these as INPUT variables as well so that users can just change the values of variables in the Shell script without having to go through the rest of the code to modify.

Reply:
Removed this from the script based on my understanding that most AuthN solution generally gives export commands for the users. 

I may have missed to update the README with this requirement. Having them copy the output then set them in the script may induce human errors. I was thinking its better if they can use it on the terminal or session they are in before invoking this script.

Having this in the script not only means that the user has to update the script each time for authentication but also means they may have to copy parts of the output from AuthN command they have for each of these three. 

Rather than asking the Customer to export these variables and having an error thrown if found empty would be better, do you agree? Intention here is that users do not have to edit the script and can run it using just updated arguments. Having these long values as arguments will be a pain to manage as well.